qt-models/models.cpp

```diff
@@ -36,16 +36,29 @@ GasSelectionModel *GasSelectionModel::instance()
 	return &self;
 }
 
-//TODO: Remove this #include here when the issue below is fixed.
-#include "diveplannermodel.h"
+QStringList &GasSelectionModel::getGasList()
+{
+	static QStringList list;
+	list.clear();
```

Comment:
I understand that you are only moving the function, but...
1) In C++ you can/should return complex objects as value (RVO). This would be *much* saner if `list` was not static and returned as value.
2) Does this need to be a member function at all? Perhaps just make it a static helper local to this translation unit?